<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCommentWindow</class>
 <widget class="QMainWindow" name="SendCommentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Comments</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_hemisphere">
       <property name="text">
        <string>Hemisphere</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_hemisphere">
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unknown</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_stim_region">
       <property name="text">
        <string>Stimulus Region</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_stim_region">
       <item>
        <property name="text">
         <string>Upper Limb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Upper Arm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Forearm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hand</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower Limb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thigh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower Leg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Foot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_stim_type">
       <property name="text">
        <string>Stimulus Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_stim_type">
       <item>
        <property name="text">
         <string>Passive Movement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Active Movement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cutaneous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_construct_comment">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_construct_comment">
       <property name="text">
        <string>Construct Comment</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_extra">
       <property name="text">
        <string>Extra</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_extra"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Comment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_extra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Extra</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clear_extra</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_extra</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
